Backpack/TappableLinkLabel/Classes/BPKTappableLinkLabel.h

```diff
@@ -23,6 +23,22 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * These constants represent the different styles available for BPKTappableLinkLabels
+ */
+typedef NS_SWIFT_NAME(TappableLinkLabelStyle) NS_ENUM(NSUInteger, BPKTappableLinkLabelStyle){
+
+    /**
+     * Blue links within dark-gray labels
+     */
+    BPKTappableLinkLabelStyleDefault,
+
+    /**
+     * White bold links within white labels
```

Comment:
Should mention that this is suitable for use on blocks of blue and other solid colors